src/pip/_internal/pep425tags.py

```diff
@@ -49,6 +49,24 @@ def get_abbr_impl():
     return pyimpl
 
 
+def version_info_to_nodot(version_info):
+    # type: (Tuple[int, ...]) -> str
+    # Only use up to the first two numbers.
+    return ''.join(map(str, version_info[:2]))
+
+
+def version_info_to_versions(version_info):
```

Comment:
I'm mildly convinced by the need of `version_info_to_versions` & `version_info_to_nodot` since they are currently both only used once...

Maybe only keep `version_info_to_nodot` and directly inline `[version_info_to_nodot(version_info)] if version_info else None` in `PackageFinder.create` ?

Reply:
Good point! It won't be as straightforward to test that `PackageFinder.create()` is doing that line of code correctly, but I'll do that.